manifests/modules/security/kyverno/.workshop/terraform/addon.tf

```diff
@@ -0,0 +1,25 @@
+module "kyverno" {
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints?ref=v4.32.1//modules/kubernetes-addons/kyverno"
```

Comment:
Lets avoid using the old blueprints repo, its not getting updated any more. Can we just use `helm_release` directly?

Reply:
You mean a `helm_release` resource? What do you say about using the v5 addon wrapper?

```
helm_releases = {
    prometheus-adapter = {
      description      = "A Helm chart for Kyverno"
      namespace        = "kyverno"
      create_namespace = true
      chart            = "kyverno"
      chart_version    = "3.0.0"
      repository       = "https://kyverno.github.io/kyverno/"
      ]
    }
  }

```

Reply:
v5 wrapper is fine too, just trying to avoid taking more dependencies on EOL code

Reply:
Nice, on it!